.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  IMAGE_REGISTRY: quay.io
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
Hey can we use the latest version, but set the version explicitly:

In this case the latest available version for ubuntu is ubuntu-24.04, which may you expect to have setting that as ubuntu-latest but from https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories this is not the case.

So to be sure about what we are using, and what we can expect, best set the version

Reply:
updated